DSCResources/xRunbookDirectory/xRunbookDirectory.schema.mof

```diff
@@ -0,0 +1,11 @@
+
+[ClassVersion("1.0.0.0"), FriendlyName("RunbookDirectory")]
+class xRunbookDirectory : OMI_BaseResource
+{
+    [Key] String RunbookPath;
+    [Write] Boolean Publish;
+    [Read] Boolean Matches;

```

Comment:
You should include description.  This makes it easier to use the resource.  Especially in ISE.